app/models/neighbour_response.rb

```diff
@@ -15,6 +15,8 @@ class NeighbourResponse < ApplicationRecord
   enum(summary_tag: {supportive: "supportive", neutral: "neutral", objection: "objection"})
 
   scope :redacted, -> { where.not(redacted_response: "") }
+  scope :with_tags, -> { where.not("tags = '[]'") }
+  scope :without_tags, -> { where("tags = '[]'") }
```

Comment:
I think you can just use the hash syntax here - Active Record should do the typecasting to JSON for you, e.g.

``` ruby
scope :with_tags, -> { where.not(tags: []) }
scope :without_tags, -> { where(tags: []) }
```

Reply:
That doesn't work but using the JSON does 🤷‍♀️

Reply:
Well, this feels like a bug to me 😂 

```
irb(main):001> NeighbourResponse.where.not(tags: []).to_sql
=> "SELECT \"neighbour_responses\".* FROM \"neighbour_responses\" WHERE 1=1"
irb(main):002> NeighbourResponse.where(tags: []).to_sql
=> "SELECT \"neighbour_responses\".* FROM \"neighbour_responses\" WHERE 1=0"
```